EIPS/eip-future_salaries.md

```diff
@@ -31,7 +31,9 @@ Paradoxically, it will directly benefit miners/validators, see the discussion.
 
 `DefaultDAOInterface` = `TBD`
 
-Prior to `FORK_BLOCK_NUMBER`, [SalaryWithDAO](https://github.com/vporton/future-contracts/blob/master/contracts/SalaryWithDAO.sol) and [DefaultDAOInterface](https://github.com/vporton/future-contracts/blob/master/contracts/DefaultDAOInterface.sol) contracts will be deployed to the network and exist at the above specified addresses.
+[The contracts source (draft).](https://github.com/ethereum/EIPs/tree/master/assets/eip-3267/contracts)
```

Comment:
```suggestion
[The contracts source](../assets/eip-3267/contracts)
```

Reply:
Hmm, I don't know if linking to a *folder* works in all environments (like https://eips.ethereum.org).  Can the solidity files be flattened into one file?  That would make things easier as you could just link to that one file.

Reply:
I think flattening into one (in fact two: we have two contracts) file would make things harder to review, because the task of the auditors would be complicated because it would lose modularity.

I advise to leave it a link to the folder during the review, later we can flatten it. Also, I think, it's OK for https://eips.ethereum.org to link to a folder under https://github.com/ethereum/EIPs